index.js

```diff
@@ -396,6 +412,7 @@ var updateView = utils.sequentialize(mainQueue, function (view) {
       view.sourceDB.changes({
         conflicts: true,
         include_docs: true,
+        style: 'all_docs',
```

Comment:
^ we need to fetch all the leafs; this adds a bit more in-memory work but still saves us time overall because we can use the gen-1 optimization